source/_integrations/iammeter.markdown

```diff
@@ -13,12 +13,14 @@ ha_platforms:
   - sensor
 ---
 
-`iammeter` provides real-time readings of single-phase (WEM3080, WEM3162) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com) over Wi-Fi.
+`IAMMETER` provides real-time readings of single-phase (WEM3080) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com/docs/homeassistant) over Wi-Fi.
+[Monitor your solar PV system with IAMMETER energy monitor in Home Assistant](https://community.home-assistant.io/t/monitor-your-solar-pv-system-in-home-assistant-energy-dashboard-vs-energy-management/342710)
 
 Example Lovelace Dashboard:
 
 <p class='img'>
-  <img src='/images/integrations/iammeter/demo.jpg' />
+  ![IAMMETER energy monitor in Home Assistant](https://iammeterglobal.oss-ap-southeast-1.aliyuncs.com/img/image-20211108143148254.png)
+  ![monitor the solar pv system in Home Assistant](https://iammeterglobal.oss-ap-southeast-1.aliyuncs.com/img/image-20211108143245427.png)
```

Comment:
images need to be hosted from our repository, not external. This is for privacy reasons.

Reply:
Already done. Thanks